src/coreclr/tools/Common/Compiler/ObjectWriter/ObjectWriter.cs

```diff
@@ -591,6 +579,22 @@ public void EmitObject(Stream outputFileStream, IReadOnlyCollection<DependencyNo
             }
         }
 
+        private protected virtual void EmitSymbolRangeDefinition(string rangeNodeName, string startNodeName, string endNodeName)
+        {
+            if (!_definedSymbols.TryGetValue(startNodeName, out var startSymbol)
+                    || !_definedSymbols.TryGetValue(endNodeName, out var endSymbol))
+            {
+                throw new InvalidOperationException("The symbols defined by a symbol range must be emitted into the same object.");
```

Comment:
I have seen `string.Format(SR.Error..., format)` used in one of the files before